backends/advanced/src/advanced_omi_backend/workers/transcription_jobs.py

```
@@ -208,20 +208,23 @@ async def transcribe_full_audio_job(
         logger.info(
             f"📦 Reconstructed audio from MongoDB chunks: " f"{len(wav_data) / 1024 / 1024:.2f} MB"
         )
+    except ValueError as e:
+        # No chunks found for conversation
+        raise FileNotFoundError(f"No audio chunks found for conversation {conversation_id}: {e}")
```

Ruff (0.14.14)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)